ocaml/xapi/xapi_db_upgrade.ml

```diff
@@ -868,6 +868,18 @@ let remove_legacy_ssl_support =
       )
   }
 
+let upgrade_last_updated_field =
+  {
+    description= "update Host.last_software_update field"
+  ; version= (fun _ -> true)
+  ; fn=
+      (fun ~__context ->
+        let host = Helpers.get_localhost ~__context in
+        Db.Host.set_last_software_update ~__context ~self:host
+          ~value:(Xapi_host.get_servertime ~__context ~host)
```

Comment:
This might be misleading: we claim we've updated the host "now", when in fact we probably just installed an ISO, which could've been months/years out of date.

What is the semantics of this field:
* does it mean that the host has the latest cloud updates installed as of <timestamp>
* the date the last software update got installed was <timestamp>, but we don't know how out-of-date that is with the cloud

It might be useful to have some kind of field that tells you the latest HCP update number that got installed as well to disambiguate.

Either way I'd suggest to try to set this in some way to the "ISO" build date, e.g. can we get that based on the DBV, or some other ISO metadata?